### ISSUE TITLE
[1.16.1] Oh The Biomes You'll Go

### ISSUE THREAD

**Issue by Leorgito** [CyanideX/Unity-Modded#39]
**Mod**
Oh The Biomes You'll Go

**Download**
https://www.curseforge.com/minecraft/mc-mods/oh-the-biomes-youll-go

**Comment by CyanideX** [comment]
Does this add its own assets? I thought it was just using vanilla blocks.

**Comment by Leorgito** [comment]
> Does this add its own assets? I thought it was just using vanilla blocks.

Yeah. This mod add new flowers, blocks, types of wood etc. It would be great if it is compatible with the texture pack!

**Comment by CyanideX** [comment]
I have moved it to the suggestion list and will try to get to it once all of the todos are completed! 😀
